packages/react-ui/src/app/routes/platform/settings/workers/index.tsx

```diff
@@ -0,0 +1,264 @@
+              );
+            },
+          },
+          {
+            accessorKey: 'status',
+            header: ({ column }) => (
+              <DataTableColumnHeader column={column} title={t('Status')} />
+            ),
+            cell: ({ row }) => {
+              const status = row.original.status;
+              return (
+                <div className="flex gap-1 items-center p-2 capitalize">
+                  {status === 'ONLINE' ? (
+                    <Server size={14} className="text-green-500" />
+                  ) : (
+                    <ServerOff className="text-red-500" />
+                  )}
+                  {t(status.toLowerCase())}
+                </div>
+              );
+            },
+          },
+          {
+            accessorKey: 'information.cpuUsagePercentage',
+            header: ({ column }) => (
+              <DataTableColumnHeader column={column} title={t('CPU Usage')} />
+            ),
+            cell: ({ row }) => {
+              return (
+                <div className="flex items-center">
+                  <CircleProgress
+                    value={row.original.information.cpuUsagePercentage}
+                  />
+                </div>
+              );
+            },
+          },
+
+          {
+            accessorKey: 'information.diskInfo.percentage',
+            header: ({ column }) => (
+              <DataTableColumnHeader column={column} title={t('Disk Usage')} />
+            ),
+            cell: ({ row }) => {
+              return (
+                <div className="flex items-center">
+                  <CircleProgress
+                    value={row.original.information.diskInfo.percentage}
+                  />
+                </div>
+              );
+            },
+          },
+          {
+            accessorKey: 'information.ramUsagePercentage',
+            header: ({ column }) => (
+              <DataTableColumnHeader column={column} title={t('RAM Usage')} />
+            ),
+            cell: ({ row }) => {
+              const ramUsage = row.original.information.ramUsagePercentage;
+              const totalRam =
+                row.original.information.totalAvailableRamInBytes;
+              const formattedTotalRam = `${(totalRam / 1024 ** 3).toFixed(
+                1
+              )} GB`;
+              return (
+                <div className="flex items-center">
+                  <CircleProgress value={ramUsage} label={formattedTotalRam} />
+                </div>
+              );
+            },
+          },
+
+          {
+            accessorKey: 'updated',
+            header: ({ column }) => (
+              <DataTableColumnHeader
+                column={column}
+                title={t('Last Contact')}
+              />
+            ),
+            cell: ({ row }) => {
+              return (
+                <div className="text-start">
+                  {new Date(row.original.updated).toLocaleString()}
+                </div>
```

Comment:
{formatUtils.formatDateOnly(new Date(row.original.updated))}

```suggestion
                  {formatUtils.formatDateOnly(new Date(row.original.updated))}
                </div>
```